server/src/main/java/com/cloud/resource/ResourceManagerImpl.java

```diff
@@ -2451,11 +2452,11 @@ public HostVO fillRoutingHostVO(final HostVO host, final StartupRoutingCommand s
     @Override
     public void deleteRoutingHost(final HostVO host, final boolean isForced, final boolean forceDestroyStorage) throws UnableDeleteHostException {
         if (host.getType() != Host.Type.Routing) {
-            throw new CloudRuntimeException("Non-Routing host gets in deleteRoutingHost, id is " + host.getId());
+            throw new CloudRuntimeException(String.format("Non-Routing host gets in deleteRoutingHost, id is %s", host.getId()));
         }
 
         if (s_logger.isDebugEnabled()) {
-            s_logger.debug("Deleting Host: " + host.getId() + " Guid:" + host.getGuid());
+            s_logger.debug(String.format("Deleting Host: %s", host));
```

Comment:
Similar to VM's `toString` would occur here.

Reply:
Updated